executor/src/witgen/jit/block_machine_processor.rs

```diff
@@ -336,15 +339,12 @@ fn machine_calls<T: FieldElement>(
     e: &Effect<T, Variable>,
```

Comment:
The comment here and the comment of `completed_rows_for_fus_send` should be updated. As the code is here now, it should be identity IDs, not bus IDs.

Reply:
Hmm, they are talking about "bus send ID" and "bus identity", which is both correct, no? I'll use the term "bus send identity ID", maybe that clears up some confusion.